Makefile

```diff
@@ -167,6 +169,7 @@ clone: clean-temp
 
 .PHONY: primazactl
 primazactl: ## Setup virtual environment
+	echo '__version__ = "$(VERSION)"' > $(VERSION_FILE)
```

Comment:
should we default VERSION to `git describe --tags --always --abbrev=8 --dirty`?

this would produce the following outputs:
* `402458eb` if current commit is not tagged the first 8 digits of its SHA are used
* `402458eb-dirty` if current commit is not tagged and versioned files are locally changed
* `v0.1.0` if current commit is tagged as v0.1.0
* `v0.1.0-dirty` if current commit is tagged as v0.1.0 and versioned files are locally changed
* `v0.1.0-1-gf97b02aa` if current commit is 1 commit away from tag `v0.1.0`
* `v0.1.0-2-g402458eb` if current commit is 2 commit away from tag `v0.1.0`
* `v0.1.0-1-gf97b02aa-dirty` if current commit is 1 commit away from tag `v0.1.0` and versioned files are locally changed